central/views/imagecve/view_test.go

```diff
@@ -1390,8 +1383,22 @@ func testImages() []*storage.Image {
 	}
 }
 
-func getTestComponentID(testComponent *storage.EmbeddedImageScanComponent, imageID string) string {
-	id, _ := scancomponent.ComponentIDV2(testComponent, imageID)
-
-	return id
+func assertResponsesAreEqual(t *testing.T, expected []CveCore, actual []CveCore, testOrder bool) {
+	if !testOrder {
+		sort.SliceStable(expected, func(i, j int) bool {
+			return expected[i].GetCVE() < expected[j].GetCVE()
+		})
+		sort.SliceStable(actual, func(i, j int) bool {
+			return actual[i].GetCVE() < actual[j].GetCVE()
+		})
+	}
+	for i, flatCVE := range actual {
```

Comment:
I think we need to check if len is the same
```suggestion
	assert.Len(t, actual, len(expected)
```